### PR TITLE
Add bio for a GitHub student

### PR DESCRIPTION
**Pull request by MaryWheeler**
This was request for issue #5

### CODE REVIEW

File: mary-test.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Test file ##
+
+GitHub for Everyone
+- Learning Github Workflow
+- Creating a test file
```
Comment (MaryWheeler) [inline mary-test.md]: Test edit.

File: mary-test.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Test file ##
+
+GitHub for Everyone
+- Learning Github Workflow
+Test
```
Comment (MaryWheeler) [inline mary-test.md]: Good test :+1:

**Comment by courtlandia-zz** [comment]
looks good!

**Comment by kjnovak** [comment]
How many times can we use "test" today?

File: mary-test.md (hunk 2 above)
Comment (kjnovak) [inline mary-test.md]: glad you like your own work :wink:

**Pull request merged**